app/javascript/packs/base.jsx

```diff
@@ -62,17 +70,15 @@ function getPageEntries() {
   });
 }
 
+/**
+ * Initializes the left hand side hamburger menu
+ */
 function initializeNav() {
   const { currentPage } = document.getElementById('page-content').dataset;
-  const menuTriggers = [
-    ...document.querySelectorAll(
-      '.js-hamburger-trigger, .hamburger a:not(.js-nav-more-trigger)',
-    ),
-  ];
-  const moreMenus = [...document.getElementsByClassName('js-nav-more-trigger')];
```

Comment:
We no longer have a "more" button in the navigation, we had removed this functionality a while back but this code was left behind. There is no longer a `js-nav-more-trigger` in the code.